vitest.config.ts

```diff
@@ -15,6 +15,7 @@ export default defineConfig({
     include: ['src/**/*.spec.{js,jsx,ts,tsx}'],
     globals: true,
     environment: 'jsdom',
+    setupFiles: 'vitest.setup.ts',
```

Comment:
Does we need this?

Reply:
Yes. We currently need this for `toBeInTheDocument()` which is used frequently in multiple test files. In future we may need this to support methods from other external libraries.